### ISSUE TITLE
Static OSX binary?

### ISSUE THREAD

**Issue by ssadedin** [brentp/somalier#68]
We deploy things with linux but most developers are using OSX. It would be very convenient to have a somalier binary compiled for OSX. 

I tried to do it myself but it invoked quite a large chain of installs to bring a lot of the nim ecosystem onto my laptop and then it failed with a linker error. 

Would be happy to try and sort that out but the issues might be specific to me (old XCode, etc) - I feel like getting a binary available for OSX might be a more generally useful option for others in the same position.

**Comment by ssadedin** [comment]
NB: I see that the linker error I got was identical to this:

https://github.com/brentp/slivar/issues/13

So I guess perhaps this request is probably dependent on sorting that out.

**Comment by brentp** [comment]
excellent!
out of curiosity, Are you, or someone you're working with doing a lot of genomics processing on Mac?

**Comment by andrewpatto** [comment]
No, just needing to do some dev work - and wanted to be able to run some simple test cases on my Mac dev machine. I guess I probably should have just been able to use the docker image

**Comment by nuin** [comment]
@brentp Just saw this issue, coming from a sliver issue, and yes, I do lots of genomic processing in Macs.
